src/Sprites/ImageHeader.cpp

```diff
@@ -0,0 +1,31 @@
+#include "ImageHeader.h"
+#include <string>
+#include <stdexcept>
+#include <algorithm>
+
+const std::array<uint16_t, 6> ImageHeader::validBitCounts{ 1, 4, 8, 16, 24, 32 };
+const std::array<uint16_t, 3> ImageHeader::indexedBitCounts{ 1, 4, 8 };
+
+bool ImageHeader::IsValidBitCount(uint16_t bitCount)
+{
+	return std::find(validBitCounts.begin(), validBitCounts.end(), bitCount) != validBitCounts.end();
+}
+
+void ImageHeader::CheckValidBitCount(uint16_t bitCount)
```

Comment:
I would standardize to using Is for checks that do not throw. To me the word check is ambiguous on whether it will throw or just return a boolean, whereas is is very clear. 

I was going back and forth on using Check or Verify for a function that throws if the check is not satisfied. Check is less syllables and a smaller word which appealed to me. If you would prefer Verify, then I can go with that.

-Brett

Reply:
I like that "Check" is shorter, but you're right it is more ambiguous. I often see "Check" methods that return a boolean, so I wouldn't know without seeing the actual function. As such, I would prefer using "Verify" for exception raising methods.

Agreed on using "Is".

Reply:
Okay, Let's go with Verify then and see how it goes over Check. I'll update this branch at some point to this new syntax.